commands/init.go

```diff
@@ -158,7 +161,37 @@ func InitWebSocketClient() (*model.WebSocketClient, error) {
 	return client, nil
 }
 
+func checkValidSocket(socketPath string) error {
+	// check file mode and permissions
+	fi, err := os.Stat(socketPath)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: maybe we can add more context for the absence of the file:

```suggestion
	if err != nil && os.IsNotExist(err) {
		return fmt.Errorf("check the server configuration for local-mode, the socket file does not exist: %s", socketPath)
	} else if err != nil {
		return err
	}
	
```

Reply:
Makes sense, added!! Thanks!! :)